blaze/expr/arithmetic.py

```diff
@@ -186,7 +201,8 @@ class Div(Arithmetic):
 
     @property
     def _dtype(self):
-        lhs, rhs = discover(self.lhs).measure, discover(self.rhs).measure
+        lhs = discover(self._operands[0]).measure, 
+        rhs = discover(self._operands[1]).measure
```

Comment:
revert the above two lines, they aren't necessary because you've already defined `self.lhs` and `self.rhs`